pawn/tf2items_manager.sp

```diff
@@ -1,38 +1,34 @@
 #pragma semicolon 1 // Force strict semicolon mode.
-
+#pragma newdecls required // Force new syntax
 // ====[ INCLUDES ]====================================================
 #include <sourcemod>
 #define REQUIRE_EXTENSIONS
 #include <tf2items>
 
 // ====[ CONSTANTS ]===================================================
-#define PLUGIN_NAME		"[TF2Items] Manager"
-#define PLUGIN_AUTHOR		"Damizean & Asherkin"
-#define PLUGIN_VERSION		"1.4.3"
-#define PLUGIN_CONTACT		"http://limetech.org/"
+#define PLUGIN_NAME "[TF2Items] Manager"
+#define PLUGIN_AUTHOR "Damizean & Asherkin"
+#define PLUGIN_VERSION "1.4.4"
+#define PLUGIN_CONTACT "http://limetech.org/"
 
-#define ARRAY_SIZE			2
-#define ARRAY_ITEM			0
-#define ARRAY_FLAGS		1
+#define ARRAY_SIZE 2
+#define ARRAY_ITEM 0
+#define ARRAY_FLAGS 1
 
 //#define DEBUG
 
 // ====[ VARIABLES ]===================================================
-new Handle:g_hPlayerInfo;
-new Handle:g_hPlayerArray;
-new Handle:g_hGlobalSettings;
-new Handle:g_hCvarEnabled;
-new bool:g_bPlayerEnabled[MAXPLAYERS + 1] = { true, ... };
-new Handle:g_hCvarPlayerControlEnabled;
+Handle g_hPlayerInfo, g_hPlayerArray, g_hGlobalSettings;

```

Comment:
Prefer one variable definition per line (mainly because of confusing syntax in other languages that I think SourcePawn avoids, but I have it hard-coded as bad style).